protocol-units/da-sequencer/node/src/lib.rs

```diff
@@ -51,8 +53,22 @@ pub async fn start(mut dot_movement: dot_movement::DotMovement) -> Result<(), an
 	let (request_tx, request_rx) = mpsc::channel(GRPC_REQUEST_CHANNEL_SIZE);
 	// Start gprc server
 	let grpc_address = da_sequencer_config.grpc_listen_address;
-	let grpc_jh =
-		tokio::spawn(async move { run_server(grpc_address, request_tx, whitelist).await });
+	let verifying_key = da_sequencer_config
+		.main_node_verifying_key
+		.clone()
+		.map(|str| {
+			<[u8; 32]>::from_hex(str)
+				.map_err(|_| anyhow::anyhow!("Invalid main_node_verifying_key hex"))
+				.and_then(|hex| {
+					VerifyingKey::from_bytes(&hex)
+						.map_err(|_| anyhow::anyhow!("Invalid main_node_verifying_key key"))
+				})
+		})
+		.transpose()?;
```

Comment:
Seems quite unwieldily just to get the verify_key. If we expect this `verifying_key` to be accessed elsewhere, abstract the map() call into some method on `main_node_verifying_key`.